pkg/operator/controller/machineconfig/constants.go

```diff
@@ -41,6 +41,10 @@ const (
 	CrioUnixSocketConfData = `[Service]
 Environment="ENABLE_PROFILE_UNIX_SOCKET=true"`
 
+	// CrioUnixSocketEnvString refers to the environment variable info
+	// string that helps in finding if the profiling is enabled by default
+	CrioUnixSocketEnvString = "ENABLE_PROFILE_UNIX_SOCKET=true"
```

Comment:
This variable can be used in the conf file data (variable just above). Example: https://go.dev/play/p/2WYvJqUgekV

Reply:
The only reason I didn't make this change is to maintain it as a constant instead of a variable. WDYT?

Reply:
Once both variables are tight together, we make a link between 2 controllers and this will help us to not forget to update both when the time will come to think about scheduling on other node types (not only workers). We can sacrifice 1 compile time substitution for that.